v1.0/v1.0/bwa-mem-tool.cwl

```diff
@@ -5,8 +5,6 @@ cwlVersion: v1.0
 class: CommandLineTool
 
 hints:
-  - class: ResourceRequirement
-    coresMin: 4
```

Comment:
@StarvingMarvin What guaranties would we have that the data from resource requirements is actually honored?

Should we make a new category of tests, of jobs that should fail? Then we could make some ridiculous requirement for a simple tool, like `coresMin: 9000`, and be more certain that a cwl runner is taking the requirement into account.